<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D&amp;E's Watercolor Stencil Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(60, 81, 114)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Color1Button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(18,41,74);

border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Color2Button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(220, 36, 28);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Color3Button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(240, 219, 41);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Color4Button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(0, 125, 92);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Color5Button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(0, 221, 212);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Color6Button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(242, 180, 53);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Color7Button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(25, 25, 25);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>784</width>
           <height>448</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>556</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:White;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="ImageLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background:rgb(82, 110, 156);
border:none;</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionReloadColors"/>
   <addaction name="separator"/>
   <addaction name="actionChoose_Colours"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons8-new-64.png</normaloff>:/new/icons/icons8-new-64.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons8-image-file-64.png</normaloff>:/new/icons/icons8-image-file-64.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionReloadColors">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons8-synchronize-64.png</normaloff>:/new/icons/icons8-synchronize-64.png</iconset>
   </property>
   <property name="text">
    <string>ReloadColors</string>
   </property>
   <property name="toolTip">
    <string>Reload Colors</string>
   </property>
  </action>
  <action name="actionChoose_Colours">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons8-rainbow-64.png</normaloff>:/new/icons/icons8-rainbow-64.png</iconset>
   </property>
   <property name="text">
    <string>Choose Colours</string>
   </property>
   <property name="toolTip">
    <string>Choose Colours</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/icons8-money-box-64.png</normaloff>:/new/icons/icons8-money-box-64.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
